rest/api_test.go

```diff
@@ -7719,3 +7719,114 @@ func TestDocChannelSetPruning(t *testing.T) {
 	assert.Equal(t, uint64(1), syncData.ChannelSetHistory[0].Start)
 	assert.Equal(t, uint64(12), syncData.ChannelSetHistory[0].End)
 }
+
+func TestAdminAPIAuth(t *testing.T) {
```

Comment:
Should this belong in `admin_api_test.go` or `admin_api_auth_test.go` if we anticipate more to be written.

```suggestion
func TestAdminAPIAuthNoAccess(t *testing.T) {
```

Reply:
Not renamed test yet as it'll likely include non-noaccess perms but point taken.